### ISSUE TITLE
first_frame_gt and cls_gt value 0 during training

### ISSUE THREAD

**Issue by Hartwell-327** [hkchengrex/Cutie#141]
I encountered the first_frame_gt and cls_gt value 0 during training, my data set path is correct, rgb has a value, can read the picture, please how to solve the problem

**Comment by Hartwell-327** [comment]
![Image](https://github.com/user-attachments/assets/db9e874b-7643-4044-baed-3d55e088e582)

**Comment by hkchengrex** [comment]
Does training work if you use our default dataset and code?

**Comment by Hartwell-327** [comment]
I used theDAVIS dataset, turned off pre-training, the code changed distributed to single GPU, and I don’t know if it was the reason Distributed Sampler replaced by RandomSampler.

**Comment by hkchengrex** [comment]
There might be a problem with how you changed distributed training to single-GPU training. Technically, you do not need to change any code -- distributed training will work with a single GPU.

**Comment by Hartwell-327** [comment]
Sorry, I did not reply in time, I have solved this problem and started training, but in the middle and late stages of training, there will be a situation of loss in the middle and late stages of training, this will happen to replace my data set, and there is a value of loss after changing the parameters to reduce lr, but the total loss does not drop during the training process.

**Comment by Hartwell-327** [comment]
As I read on  blogs, other bloggers will have Nan situations when they try to train.

**Comment by hkchengrex** [comment]
What blog?

Can you try to replace this line:
https://github.com/hkchengrex/Cutie/blob/ec5cdd4cf16f75c73ad785a2f96fb97dbad4125a/cutie/model/trainer.py#L64

with 
```python
self.scaler = torch.amp.GradScaler('cuda', enabled=False)
```

?

**Comment by Hartwell-327** [comment]
Thank you for your reply, the blog is my search on CSDN related to see, I closed the AMP after there is a relief problem, I use our dataset training, the model can not learn anything, in Cutie-base-mega.pth as a pre-training, my dataset as the main training to improve the effect,Please advise, the interactive config in the project-“gui_config.yaml”, I see two model paths, this train.py get is only the replacement of Cutie-base-mega.pth model, Is there a training script for the ritm weight： coco-lvis-h18.pth replacement model?

**Comment by Hartwell-327** [comment]
What I want to do is medically related experiments, my data set needs to be divided in the object is relatively fine, the current model segmentation effect will be thicker than the actual object, the edge is also rough.Resulting in my test results, such as dice score and f1 score are low, like trying to change the ritm weight to see if there is any improvement, thank you for your help.

**Comment by Hartwell-327** [comment]
Sorry, I didn't see you've already given the link. I have found in the current experiment that the value of the mask image after the interaction is not necessarily better than the effect of the direct verification with the model, and will the situation in your experiment, such as the mask after the evaluation run and the interaction mask comparison, the former has a higher value of the result value?

**Comment by hkchengrex** [comment]
I am not sure if I understand your question. Could you elaborate?

**Comment by Hartwell-327** [comment]
Sorry I didn’t make it clear that after I tried to train with my dataset, the result in eval.py was higher than the f1 value of the result after my interaction. Is there such a situation in your experiment, and I guess it is because the first frame mask was given during the test reasoning process.

**Comment by hkchengrex** [comment]
By interaction, do you mean training? If the model was not well trained, the performance could degrade.

**Comment by Hartwell-327** [comment]
No, I'm referring to the part that trains to split inference, and I tried the mask that I got directly with "python cutie/eval_vos.py dataset=[dataset] weights=[path to model file] model=[small/base]" to do the f1 value of the indicator evaluation is higher than the f1 value of the mask I got in the interactive gui. So I guess if this is the case because the eval.py process gave the first frame mask.
